### ISSUE TITLE
Sandworm (sandstorm_tremor) and Ignis Altar (dinos_ignis_altar) are not affected by Master Architects (supreme_architects_system)

### ISSUE THREAD

**Issue by Mmlh1** [cigumo/krvi#485]

**Comment by pablorealini** [comment]
I think we currently have two big issues here. One is that towers with very low damage will not be affected at all since a 10% increase may not be enough to add at least 1 extra damage point. We can change the upgrade to be 10% or at least 1 damage point… but even with this fix, some levels for both towers will still have no effect.

At this stage of the game, I don’t think it’s safe to change the behavior of the upgrade. I will just add Supreme to both towers and accept that not all levels will be affected.

These are not the only towers with this issue.

**Comment by Mmlh1** [comment]
Yes, I agree it will not change all levels. The main reason it's impactful is that it stacks with Murglun and Oloch tower buffs, so it is actually observable in the game. Just making them affected without further changes therefore sounds great!

**Comment by Mmlh1** [comment]
Btw this combines with #495, so it might still not quite work correctly with Ignis if that issue is not resolved
